user-docs/documentation/docs/how-to-guides/5-deploy-cdk-constructs.md

```diff
@@ -0,0 +1,220 @@
+
+  plugins: [
+    ...
+    '@swarmion/serverless-cdk-plugin',
+    ...
+  ],
+
+  provider: {...},
+  functions: {
+    myLambda: {
+      ...
+      environment: {
+        TABLE_NAME: getCdkProperty('dynamodbName'),
+        TABLE_ARN: getCdkProperty('dynamodbArn'),
+      },
+      ...
+    },
+  },
+  resources: {...},
+  construct: MyConstruct,
+};
+```
+
+## Export information from serverless to CDK
+
+### Using the serverless props
+
+The plugin exposes a construct wrapper to access the config file and the runtime serverless service object if you need it. The example below shows how to retrieve the stage value.
+
+```typescript
+import ServerlessCdkPlugin, {
+  ServerlessProps,
+  ServerlessCdkPluginConfig,
+} from '@swarmion/serverless-cdk-plugin';
+
+
+class MyConstruct extends ServerlessCdkPlugin.ServerlessConstruct {
+  constructor(scope: Construct, id: string, serverlessProps: ServerlessProps) {
+    super(scope, id);
+
+    new Table(this, 'MyDynamodbTable', {
+      tableName: `table-${serverlessProps.service.provider.stage}`,
+      partitionKey: { name: 'PK', type: AttributeType.STRING },
+      sortKey: { name: 'SK', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      removalPolicy:
+        serverlessProps.service.provider.stage === 'production'
+          ? RemovalPolicy.RETAIN
+          : RemovalPolicy.DESTROY,
+    });
+  }
+}
+
+const serverlessConfiguration: AWS & ServerlessCdkPluginConfig = {
+  service: "my-cdk-construct-service",
+
+  plugins: [
+    ...
+    '@swarmion/serverless-cdk-plugin',
+    ...
+  ],
+
+  provider: {...},
+  functions: {...},
+  resources: {...},
+  construct: MyConstruct, // You can provide either a Construct or a ServerlessConstruct here
+};
+```
+
+### Using serverless variables
+
+You can also use serverless variables directly in the CDK.
+
+```typescript
+class MyConstruct extends Construct {
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+
+    new Table(this, 'MyDynamodbTable', {
+      tableName: 'table-${sls:stage}', // Use variables as if you were in the serverless config
+      partitionKey: { name: 'PK', type: AttributeType.STRING },
+      sortKey: { name: 'SK', type: AttributeType.STRING },
+      billingMode: BillingMode.PAY_PER_REQUEST,
+    });
+  }
+}
+```
+
+:::warning
+For implementation reasons, to use serverless variables, you need to use a cdk plugin variable at least once in your configuration.
+
+If you use the `getCdkPropertyHelper` in your config, you're all set
+
+You can also add `'${serverlessCdkBridgePlugin:magicValue}'` to any `custom` key of your serverless config.
+:::
+
+## Deploy multiple CDK constructs
+
+If you need to deploy multiple constructs, you can follow the example below which defines two Dynamodb tables.
+
+## CDK caveats
+
+### CDK naming limitations
+
+Some CDK constructs sanity check properties, and may reject names containing serverless variables because they will not be resolved yet.
```

Comment:
I would put this directly in the `### Using serverless variables` section, since it is directly related.
And inside a `caution` admonition